app/assets/javascripts/oxalis/workers/readme.md

```diff
@@ -0,0 +1,33 @@
+# Web Worker Modules
```

Comment:
Wow, awesome that you took the time to write this explanation, extremely helpful and easy to understand! :1st_place_medal: